ui/page/governance/governance_page.go

```diff
@@ -117,6 +124,24 @@ func (pg *Page) Layout(gtx C) D {
 	})
 }
 
+func (pg *Page) layoutMobile(gtx layout.Context) layout.Dimensions {
+	if !pg.isGovernanceFeatureEnabled() {
+		return components.UniformMobile(gtx, false, pg.splashScreenLayout)
+	}
+	return components.UniformMobile(gtx, true, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(pg.layoutPageTopNav),
+			layout.Rigid(pg.layoutTabs),
+			layout.Rigid(pg.Theme.Separator().Layout),
+			layout.Flexed(1, func(gtx C) D {
+				return layout.Inset{Top: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+					return pg.CurrentPage().Layout(gtx)
+				})
+			}),
+		)
+	})
+}
```

Comment:
No need for this as i see the only use was is in the splash screen `uniformMobileLayout()`

Reply:
we should leave as is, further details here https://github.com/planetdecred/godcr/pull/971#discussion_r904149266